src/libsync/propagateupload.cpp

```diff
@@ -89,17 +89,14 @@ void PUTFileJob::start()
     AbstractNetworkJob::start();
 }
 
-bool PUTFileJob::finished()
+void PUTFileJob::finished()
```

Comment:
See above. This should probably be renamed to `PutFileJob`, maybe in a follow-up PR.